beacon_node/network/src/beacon_processor/mod.rs

```diff
@@ -92,15 +92,15 @@ pub const MAX_WORK_EVENT_QUEUE_LEN: usize = 16_384;
 const MAX_IDLE_QUEUE_LEN: usize = 16_384;
 
 /// The maximum size of the channel for re-processing work events.
-const MAX_SCHEDULED_WORK_QUEUE_LEN: usize = 3 * MAX_WORK_EVENT_QUEUE_LEN / 4;
+const MAX_SCHEDULED_WORK_QUEUE_LEN: usize = MAX_WORK_EVENT_QUEUE_LEN;
 
 /// The maximum number of queued `Attestation` objects that will be stored before we start dropping
 /// them.
-const MAX_UNAGGREGATED_ATTESTATION_QUEUE_LEN: usize = 16_384;
+const MAX_UNAGGREGATED_ATTESTATION_QUEUE_LEN: usize = 32_768;
 
-/// The maximum number of queued `Attestation` objects that will be stored before we start dropping
-/// them.
-const MAX_UNAGGREGATED_ATTESTATION_REPROCESS_QUEUE_LEN: usize = 8_192;
+/// The maximum number of queued `Attestation` objects that reference an unknown
+/// block that will be stored before we start dropping them.
+const MAX_UNAGGREGATED_ATTESTATION_REPROCESS_QUEUE_LEN: usize = 32_768;
```

Comment:
I wonder whether we should keep the reprocess queue limit at half the regular limit, i.e. only bump it to 16K.

I'd like to have some good basis for this decision (queueing theory?) but I don't, other than "if it ain't broke, don't fix it".